shell/browser/api/electron_api_base_window.cc

```diff
@@ -95,6 +94,8 @@ BaseWindow::BaseWindow(v8::Isolate* isolate,
       options, parent.IsEmpty() ? nullptr : parent->window_.get()));
   window_->AddObserver(this);
 
+  SetContentView(View::Create(isolate));
```

Comment:
Do we have an option to allow passing in a `view` to the constructor? It could be a nice way to possibly transfer a view to a new window.

Reply:
That would be only a fairly minor performance improvement over this, I think:

```js
const w = new BaseWindow
w.contentView = myView
```